Algorithms/Searching/fibonacci_search.cpp

```diff
@@ -0,0 +1,45 @@
+#include<iostream>
+using namespace std;
+void FibonacciSearch(int *a, int start, int end, int *fib, int index, int item) {
+  int i, mid;
+  mid = start+fib[index-2];
+  if(item == a[mid]) {
+      cout<<"\n item found at "<<mid<<" index.";
+      return;
+  } else if(item == a[start]) {
+      cout<<"\n item found at "<<start<<" index.";
+      return;
+    } else if(item == a[end]) {
+        cout<<"\n item found at "<<end<<" index.";
+        return;
+      } else if(mid == start || mid == end) {
+          cout<<"\nElement not found";
+          return;
+        } else if(item > a[mid])
+            FibonacciSearch(a, mid, end, fib, index-1, item);
+          else
+            FibonacciSearch(a, start, mid, fib, index-2, item);
+}
+
+int main() {
+  int n, i, fib[20], a[10]={3, 7, 55, 86, 7, 15, 26, 30, 46, 95};

```

Comment:
make it dynamic

Reply:
Made the required changed and made a new pull request